packages/typespec-client-generator-core/src/internal-utils.ts

```diff
@@ -489,8 +484,8 @@ export function twoParamsEquivalent(
  * @param parameters
  * @returns
  */
-export function isHttpBodySpread(httpBody: HttpOperationBody | HttpOperationMultipartBody) {
-  return httpBody.property === undefined;
+export function isHttpBodySpread(httpBody: HttpPayloadBody): boolean {
```

Comment:
@tadelesh it seems only this would be impacted for tcgc.

@witemple-msft I checked for current JS RLC side this change would not introduce any breakings for existing code.

/cc @allenjzhang

Reply:
for api breaking, this is the only code impacted. but we are still discussing if tcgc should fall back to bytes before language support.

Reply:
resolving given the changes to TCGC incorporated here